### PR TITLE
Replace Magento CLA with Adobe

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) replaces the Magento CLA with the Adobe CLA in the contribution guidelines.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following diagram shows the contribution workflow:
 
     **We will close your pull request if you do not respond to feedback in two weeks.**
 
-**Note:** If you have not signed the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement), the pull request provides a link. Magento requires a signed form and agreement to the terms for contribution.
+**Note:** If you have not signed the [Adobe Contributor License Agreement](https://opensource.adobe.com/cla.html), the pull request provides a link. You must sign the the CLA before can accept your contribution.
```
Comment (filmaj) [inline .github/CONTRIBUTING.md]:
```suggestion
**Note:** If you have not signed the [Adobe Contributor License Agreement](https://opensource.adobe.com/cla.html), the pull request provides a link. You must sign the CLA before we can accept your contribution.
```

**Comment by jeff-matthews** [comment]
running tests

**Comment by dshevtsov** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**